fastboot-tests/index-test.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-env node, mocha */
+'use strict';
+
+const expect = require('chai').expect;
+const setupTest = require('ember-fastboot-addon-tests').setupTest;
+
+describe('index', function() {
+  setupTest('fastboot'/*, options */);
+
+  it('does not blow up', function() {
+    return this.visit('/')
+      .then(function(res) {
+        let $ = res.jQuery;
+        let response = res.response;
+
+        // add your real tests here
+        expect(response.statusCode).to.equal(200);
+        expect($('body').length).to.equal(1);
+        expect($('h1').text().trim()).to.equal('ember-fastboot-addon-tests');
+        expect($('.ember-tooltip-base').text().trim()).to.contain('i am an inline tooltip');
```

Comment:
Are we able to use our own [test helpers](https://github.com/sir-dunxalot/ember-tooltips#test-helpers) here?